fe/fe-core/src/test/java/com/starrocks/utframe/StarRocksAssert.java

```diff
@@ -471,14 +471,14 @@ public StarRocksAssert withSingleReplicaTable(String sql) throws Exception {
         return this;
     }
 
-    public StarRocksAssert withSingleReplicaAsyncMv(String sql) throws Exception {
+    public StarRocksAssert withAsyncMvAndRefresh(String sql) throws Exception {
         try {
             StatementBase statementBase = UtFrameUtils.parseStmtWithNewParser(sql, ctx);
             if (!(statementBase instanceof CreateMaterializedViewStatement)) {
                 return this;
             }
             CreateMaterializedViewStatement createMaterializedViewStatement = (CreateMaterializedViewStatement) statementBase;
-            withAsyncMv(createMaterializedViewStatement, true, true);
+            withAsyncMv(createMaterializedViewStatement, false, true);
```

Comment:
It's more stable in ut tests if mv only has 1 replica to avoid maintaining tablet indexes.